### ISSUE TITLE
Active balancer board v4.90

### ISSUE THREAD

**Issue by emajju** [stuartpittaway/diyBMSv4#179]
Hi, 
What about preparing active balancer board compatible with v4.90 version?
I thought about using eta3000 driver so inductive type. I could prepare schematic and layout if I will change myself to kicad but I don't have any reasonable battery now for tests :(
Do you think it would be useful?

**Comment by stuartpittaway** [comment]
Yes, this could be useful, if you have time to sketch out the basic circuit, we could look at how this could work.
